release/ray_release/byod/build.py

```diff
@@ -22,6 +27,47 @@
 PYTHON_VERSION = "3.8"
 
 
+def build_anyscale_champagne_image(
```

Comment:
should we have a unit test for this function too?

Reply:
totally!

Reply:
maybe remove the word "anyscale"? I do not think it is really related to "anyscale" here.

where does the word "champagne" come from?

Reply:
champagne comes from the marketing team ;), as a replacement suggestion for dogfooding